.eslintrc.js

```diff
@@ -322,6 +322,7 @@ module.exports = {
         'x-pack/test/functional/apps/**/*.js',
         'x-pack/legacy/plugins/apm/**/*.js',
         'test/*/config.ts',
+        'test/*/config_open.ts',
```

Comment:
What do you think is better? Exclude all config files (`'test/*/config*.ts',`) or only the `config_open.ts`?

Reply:
@FrankHassanabad

Reply:
I did it that way because if we use the rule you are suggesting we are going to skip all config files. As this config files are usually written by Kibana team and we are creating a new type, I just added the same rule but with the new config file so I don't want to skip the check for more things.